test/expected/copy.out

```diff
@@ -95,3 +95,17 @@ SELECT create_hypertable('hyper2', 'time', chunk_time_interval => 10);
 (1 row)
 
 \copy hyper2 from data/copy_data.csv with csv header ;
+----------------------------------------------------------------
+-- Testing COPY TO.
+----------------------------------------------------------------
+\c :TEST_DBNAME :ROLE_SUPERUSER
+SET client_min_messages TO NOTICE;
+-- COPY TO using a hypertable will not copy any tuples, but should
```

Comment:
The comment can be shorten as it clear from the test output what is expected to happen.